embedly/embedly/api/views.py

```diff
@@ -34,8 +34,8 @@ def version():
     )
 
 
-@blueprint.route('/extract')
-def extract_urls():
+@blueprint.route('/extract', methods=['GET'])
+def extract_urls_v1():
```

Comment:
I would say we don't want to. we should turn off V1. at this point a breaking change should be OK, we're in alpha